tests/10-http.js

```diff
@@ -75,7 +75,10 @@ describe('bedrock-messages HTTP API', function() {
           brMessages.store(
             helpers.createMessage({recipient: user.identity.id}), callback);
         },
-        get: ['insert', function(callback) {
+        process: ['insert', function(callback) {
+          brMessages._cleanupJob(callback);
+        }],
+        get: ['process', function(callback) {
           request.post(
             helpers.createHttpSigRequest(
```

Comment:
`createHttpSigRequest` -- don't abbreviate unless using a well-known acronym. So `Http` is fine, but `Sig` should be spelled out.